pages/linux/vcgencmd.md

```diff
@@ -0,0 +1,28 @@
+# vcgencmd
+
+> Print system information for a Raspberry Pi.
+> More information: <https://www.raspberrypi.org/documentation/raspbian/applications/vcgencmd.md>.
+
+- List all available commands:
+
+`vcgencmd commands`
+
+- Print the current CPU temperature:
+
+`vcgencmd measure_temp`
+
+- Print the current voltage usage:
+
+`vcgencmd measure_volts`
```

Comment:
According to the docs you have to specify a specific block here (`{{core|sdram_c|sdram_i|sdram_p}}`)

Reply:
% vcgencmd measure_volts
volt=1.2000V

Reply:
Then are those maybe optional? Should  we include them?

Also, I'm not really happy with `voltage usage`, since that's technically not really correct. I'd just say `voltage`.

Reply:
Yes, voltage usage is incorrect